cfgov/jinja2/v1/_includes/atoms/large_radio.html

```diff
@@ -0,0 +1,57 @@
+{# ==========================================================================
+
+   large_radio.render()
+
+   ==========================================================================
+
+   Builds radio button fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:   Name of the field
+
+   settings.fields:   An array of all the radio button labels.
+
+   settings.size:     The size of the fields. Defaults to 1-2.
+                      Possible options: 1, 1-2, 1-3
+
+   options (optional):
+
+   options.required:  Whether the field is required. Defaults to false.
+
+   options.disabled:  Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro render(settings, options) -%}
+  {% from 'macros/util/format/url.html' import slugify as slugify %}
+  {% set id_slug = slugify( settings.legend ) %}
+  <fieldset class="u-reset">
+      <div class="form-l_col
+                  form-l_col-1
+                  u-mb0">
+          <legend class="form-label-header">
+              {{ settings.legend }}
+              {%- if options.required -%}
+                  * <span class="u-visually-hidden">Required</span>
+              {%- endif -%}
+          </legend>
+      </div>
+      {% for field in settings.fields %}
+      <div class="form-l_col
+                  form-l_col-{{ settings.size }}
+                  form-l-inset">
+              <div class="form-l-inset_container">
+                  <input class="cf-input"
+                         type="radio"
+                         name="form_{{ id_slug }}"
```

Comment:
We really need a legend for these buttons, it's a hard requirement, so we probably wouldn't need this.